src/bin/visit.c

```diff
@@ -97,6 +103,40 @@ static const char usage[] =
 "        -env                 Print environment strings set up by the launcher\n"
 "\n";
 
+// Extra usage flags for Windows (not part of visitusage.txt)
+// The Tools section will always be needed due to how they are launched on
+// Windows.
+// Some of the others are part of visitusageplus.txt so can be removed if
+// it is decided that -fullhelp should be supported on Windows.
+static const char windows_usage[] =
+"\n"
+" Windows-OS specific options.\n"
+"\n"
+"    Tools options:\n"
+"--------------------------------------------------------------------------\n"
+"        -xmledit             Run the xmledit tool.\n"
```

Comment:
Most of these are args related to development and not *use* of VisIt. Makes me think we could probably opt to re-structure our help text a bit to make this clearer and to make sure *users* can easily get only the options they are likely to care about.

Reply:
Great idea, oh great ticket writer :-)